specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -0,0 +1,1270 @@
+				"operationId": "QuotaRequest_Create",
+				"x-ms-examples": {
+					"Quotas_Request_listForCompute": {
+						"$ref": "./examples/putComputeOneSkuQuotaRequest.json"
+					}
+				},
+				"x-ms-long-running-operation": true,
+				"x-ms-long-running-operation-options": {
+					"final-state-via": "location"
+				},
+				"parameters": [
+					{
+						"$ref": "#/parameters/SubscriptionIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ProviderIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ResourceNameInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"name": "createQuotaRequest",
+						"in": "body",
+						"description": "Quota requests payload.",
+						"required": true,
+						"schema": {
+							"$ref": "#/definitions/quotaRequestOneResourceProperties"
+						}
+					},
+					{
+						"$ref": "#/parameters/IfMatchRequiredParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK. Returns Quota requests details.",
+						"schema": {
+							"$ref": "#/definitions/quotaRequestOneResourceProperties"
+						}
+					},
+					"201": {
+						"description": "The request is accepted and is being processed. Please use requestId to check the status.",
+						"schema": {
+							"$ref": "#/definitions/quotaRequestOneResourceProperties"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ExceptionResponse"
+						}
+					}
+				}
+			},
+			"patch": {
+				"tags": [
+					"QuotaRequests"
+				],
+				"summary": "Submits a Quota Request for a resource provider at the specified location for the specific resource in the parameter.",
+				"description": "Submits Quota change request for a resource provider for the specified location for the specific resource in the parameter. To use, first make a Get request to get quota information. This information consists of a list of resources and information regarding those resources. For all the resources in that list which requires an update to their quotas, update their limit fields in the response from the Get request to their new values. Then, submit individual request for each resource using the updated JSON object to this quota request API. This will update the quotas to the values specified. The AzureAsyncOperation header in the response will be used to track the status of the quota request. Please poll the url provided in AzureAsyncOperation Header to get the status.",
+				"operationId": "QuotaRequest_Update",
```

Comment:
`Quotas_Update` so that all operations under this path are grouped together.

Reply:
Updated as suggested.